data/test/index.js

```diff
@@ -38,6 +55,33 @@ describe( 'select', () => {
 		expect( select( 'reducer1', 'selector2' ) ).toEqual( 'result2' );
 		expect( selector2 ).toBeCalledWith( store.getState() );
 	} );
+
+	it( 'prints an error when a selector is called on a non-existing reducer', () => {
+		/* eslint-disable no-console */
+		const originalConsoleError = console.error;
+		console.error = jest.fn( () => {} );
+
+		expect( select( 'reducer1', 'selector1' ) ).toBeUndefined();
+		expect( console.error ).toHaveBeenCalled();
```

Comment:
Should we use https://github.com/WordPress/packages/tree/master/packages/jest-console instead?